scripts/india_rbi/below_poverty_line/preprocess.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import csv
+import pandas as pd
+import numpy as np
+import urllib.request
+
+INDIA_ISO_CODES = {
+    "Andhra Pradesh": "dcid:wikidataId/Q1159",
+    "Arunachal Pradesh": "dcid:wikidataId/Q1162",
+    "Assam": "dcid:wikidataId/Q1164",
+    "Bihar": "dcid:wikidataId/Q1165",
+    "Chattisgarh": "dcid:wikidataId/Q1168",
+    "Chhattisgarh": "dcid:wikidataId/Q1168",
+    "Goa": "dcid:wikidataId/Q1171",
+    "Gujarat": "dcid:wikidataId/Q1061",
+    "Haryana": "dcid:wikidataId/Q1174",
+    "Himachal Pradesh": "dcid:wikidataId/Q1177",
+    "Jharkhand": "dcid:wikidataId/Q1184",
+    "Karnataka": "dcid:wikidataId/Q1185",
+    "Kerala": "dcid:wikidataId/Q1186",
+    "Madhya Pradesh": "dcid:wikidataId/Q1188",
+    "Maharashtra": "dcid:wikidataId/Q1191",
+    "Manipur": "dcid:wikidataId/Q1193",
+    "Meghalaya": "dcid:wikidataId/Q1195",
+    "Mizoram": "dcid:wikidataId/Q1502",
+    "Nagaland": "dcid:wikidataId/Q1599",
+    "Odisha": "dcid:wikidataId/Q22048",
+    "Punjab": "dcid:wikidataId/Q22424",
+    "Rajasthan": "dcid:wikidataId/Q1437",
+    "Sikkim": "dcid:wikidataId/Q1505",
+    "Tamil Nadu": "dcid:wikidataId/Q1445",
+    "Telengana": "dcid:wikidataId/Q677037",
+    "Telangana": "dcid:wikidataId/Q677037",
+    "Tripura": "dcid:wikidataId/Q1363",
+    "Uttarakhand": "dcid:wikidataId/Q1499",
+    "Uttar Pradesh": "dcid:wikidataId/Q1498",
+    "West Bengal": "dcid:wikidataId/Q1356",
+    "Andaman and Nicobar Islands": "dcid:wikidataId/Q40888",
+    "Andaman & Nicobar Islands": "dcid:wikidataId/Q40888",
+    "Chandigarh": "dcid:wikidataId/Q43433",
+    "Dadra and Nagar Haveli": "dcid:wikidataId/Q46107",
+    "Dadra & Nagar Haveli": "dcid:wikidataId/Q46107",
+    "Dadar Nagar Haveli": "dcid:wikidataId/Q46107",
+    "Daman and Diu": "dcid:wikidataId/Q66710",
+    "Daman & Diu": "dcid:wikidataId/Q66710",
+    "Delhi": "dcid:wikidataId/Q1353",
+    "Jammu and Kashmir": "dcid:wikidataId/Q66278313",
+    "Jammu & Kashmir": "dcid:wikidataId/Q66278313",
+    "Ladakh": "dcid:wikidataId/Q200667",
+    "Lakshadweep": "dcid:wikidataId/Q26927",
+    "Lakshwadeep": "dcid:wikidataId/Q26927",
+    "Pondicherry": "dcid:wikidataId/Q639421",
+    "Puducherry": "dcid:wikidataId/Q639421",
```

Comment:
To make sure I understand, why are we mapping these to DCIDs? Why not isoCodes?

Reply:
I thought it would be easy. But I will change it back to isoCodes.

Reply:
Changed to use ISO codes.